webpack.config.js

```diff
@@ -266,8 +266,8 @@ const common = {
       // Images: Copy image files to build folder
       { test: /\.(?:ico|gif|png|jpg|jpeg)$/i, type: 'asset/resource' },
 
-      // Fonts and SVGs: Inline files
-      { test: /\.(woff2|svg|)$/, type: 'asset/inline' },
+      // Inline SVGs
+      { test: /\.(svg|)$/, type: 'asset/inline' },
```

Comment:
I'm wondering if the fonts are still in the dist folder. will peek before approving/merging

Reply:
Yep. But removed in the last build.